docs/v6_upgrade.md

```diff
@@ -66,21 +69,22 @@ Now you have a straight access to the Webpack configuration and you can change i
 
   ```js
   // config/webpack/base.js
-  const { webpackConfig, merge } = require('@rails/webpacker')
-  const customConfig = require('./custom')
+  const { webpackConfig, merge } = require('@rails/webpacker');
```

Comment:
It seems like the generated versions of these files have never used semicolons. Should these be removed so as not to confuse people?

Reply:
👍